pkgs/tools/misc/hexdiff/default.nix

```
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
```

```suggestion
stdenv.mkDerivation {
```

Avoid using `rec`, especially when not needed.